src/AppBundle/Controller/FormRequestController.php

```diff
@@ -83,31 +84,12 @@ public function activationAction(Request $request, FormRequest $formRequest)
             if ($form->isValid()) {
                 $em->persist($formRequest);
                 $em->flush();
-                //TODO send email to intern
-//                $message = \Swift_Message::newInstance()
-//                    ->setSubject('Hello Email')
-//                    ->setFrom('send@example.com')
-//                    ->setTo('recipient@example.com')
-//                    ->setBody(
-//                        $this->renderView(
-//                        // app/Resources/views/Emails/registration.html.twig
-//                            'Emails/registration.html.twig',
-//                            array('name' => $name)
-//                        ),
-//                        'text/html'
-//                    )
-//                    /*
-//                     * If you also want to include a plaintext version of the message
-//                    ->addPart(
-//                        $this->renderView(
-//                            'Emails/registration.txt.twig',
-//                            array('name' => $name)
-//                        ),
-//                        'text/plain'
-//                    )
-//                    */
-//                ;
-//                $this->get('mailer')->send($message);
+                $notification = new EmailNotification();
```

Comment:
$this->get('app.email_notification')->sendNotification()
that is a service, not a regular class